cmd/limactl/info.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/lima-vm/lima/pkg/limayaml"
+	"github.com/lima-vm/lima/pkg/version"
+	"github.com/spf13/cobra"
+)
+
+func newInfoCommand() *cobra.Command {
+	infoCommand := &cobra.Command{
+		Use:   "info",
+		Short: "Show diagnostic information",
+		Args:  cobra.NoArgs,
+		RunE:  infoAction,
+	}
+	return infoCommand
+}
+
+type Info struct {
+	Version         string             `json:"version"`
+	DefaultTemplate *limayaml.LimaYAML `json:"defaultTemplate"`
+	// TODO: add diagnostic info of QEMU
+}
+
+func infoAction(cmd *cobra.Command, args []string) error {
+	y, err := limayaml.Load(limayaml.DefaultTemplate, "")
+	if err != nil {
+		return err
+	}
+	info := &Info{
+		Version:         version.Version,
+		DefaultTemplate: y,
+	}
```

Comment:
I'm not really sure what the use case is for this command; when would you need the default template in JSON format? I have no objections to adding this, but without additional diagnostics included, I can't think of a use case.

Reply:
When potentially the nerdctl repo gets compromised, user can verify whether the default template is affected or not.